interface/patient_file/pos_checkout.php

```diff
@@ -231,19 +244,21 @@ function imdeleted() {
 <br>&nbsp;
 </b></p>
 </center>
+<center>
```

Comment:
U have included bootstrap but i dont see you making use of its classes. I think here a `<div class="text-center"></div>` will do if i am right. rather than `<center>` stuff